inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/source/split/MySqlSplitSerializer.java

```diff
@@ -226,6 +258,7 @@ public byte[] serialize(MySqlSplit split) throws IOException {
             writeBinlogPosition(binlogSplit.getEndingOffset(), out);
             writeFinishedSplitsInfo(binlogSplit.getFinishedSnapshotSplitInfos(), out);
             writeTableSchemas(binlogSplit.getTableSchemas(), out);
+            writeTableDdls(binlogSplit.getTableDdls(), out);
             out.writeInt(binlogSplit.getTotalFinishedSplitSize());
```

Comment:
invoke `writeTableDdls` and `readTableDdls` order could cause  compatibility issues.
Because readTableDdls will cause error when state restore for old flink job.

Reply:
done.